### DISCUSSION TITLE
Entity Framework and ordered indexes

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#58] [General]
# Entity Framework and ordered indexes

In Entity Framework 7, the team has added support for ordered indexes to the fluent API. In this blog post we will look at how to use this feature and what it means for your database.

https://steven-giesel.com/blogPost/12c2926a-cf51-438a-af5a-5d160e9f3844

**Comment by ghadzhigeorgiev** [comment]
Thanks for the reminder about existence of .IsDescending()
